app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -2,4 +2,4 @@
 
 <p>You can confirm your account email through the link below:</p>
 
-<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
+<p><%= link_to 'Confirm my account', confirmation_url(@resource, data: { keybinding: 'enter' }, confirmation_token: @token) %></p>
```

Comment:
Since this is in a mailer (part of an email sent from the application) our JavaScript that handles these won't be present. So we should probably not add this.